sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,1299 @@
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Department or organization";
+                        readonly example: "Cloud & Al Department";
+                    };
+                };
+                readonly Addresses: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Address";
+                        readonly example: "4001 1st Ave NE Redmond, WA 98052";
+                    };
+                };
+                readonly WorkPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Work phone number";
+                        readonly example: "+1 (987) 213-5674";
+                    };
+                };
+                readonly MobilePhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Mobile phone number";
+                        readonly example: "+1 (987) 123-4567";
+                    };
+                };
+                readonly Faxes: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Fax number";
+                        readonly example: "+1 (987) 312-6745";
+                    };
+                };
+                readonly OtherPhones: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "phoneNumber";
+                        readonly description: "Other phone number";
+                        readonly example: "+1 (987) 213-5673";
+                    };
+                };
+                readonly Emails: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Contact email";
+                        readonly example: "chris.smith@contoso.com";
+                    };
+                };
+                readonly Websites: {
+                    readonly type: "array";
+                    readonly items: {
+                        readonly type: "string";
+                        readonly description: "Website";
+                        readonly example: "https://www.contoso.com";
+                    };
+                };
+            };
+        };
+    };
+};
+
+// @public
+export interface CommonClientOptions extends PipelineOptions {
+    apiVersion?: FormRecognizerApiVersion;
+    stringIndexType?: StringIndexType;
```

Comment:
Also do all operations return offsets/lengths?

Reply:
Could be pretty rate but what if a customer wants to use one encoding for one operation and a different one for another?

Reply:
All operations on DocumentAnalysisClient do. Maybe I can move this to DocumentAnalysisClientOptions from common options?

To the second question, I did think about this, but it seems unlikely and creating a new client object is not so bad if you _really_ need to do that (I'm skeptical that anyone needs to change this option anyway). We could add it to both, but that would be a beta.2 item.

Reply:
> All operations on DocumentAnalysisClient do. Maybe I can move this to DocumentAnalysisClientOptions from common options?

That would be better IMO. No reason to confuse customers with options that do not apply.

>  it seems unlikely and creating a new client object is not so bad if you really need to do that (I'm skeptical that anyone needs to change this option anyway).

Fair!

Reply:
Done!